aries-site/src/components/content/PageBackground.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext } from 'grommet';
+import { useDarkMode } from '../../utils';
+
+export const PageBackground = ({ backgroundImage }) => {
+  const size = useContext(ResponsiveContext);
+  const themeMode = useDarkMode().value ? 'dark' : 'light';
+
+  return (
+    <Box
+      margin={
+        backgroundImage[size] && backgroundImage[size].margin
+          ? backgroundImage[size].margin
+          : backgroundImage.margin
+      }
+      pad={
+        backgroundImage[size] && backgroundImage[size].pad
+          ? backgroundImage[size].pad
+          : backgroundImage.pad
+      }
+      responsive={false}
+      width={size !== 'small' ? '65%' : 'fill'}
```

Comment:
65% sounds random, where is this value coming from?

Reply:
Same as below, trying to just align as closely to how the designs overlap the text.